tests/apps/service_invocation/service.yaml

```diff
@@ -0,0 +1,45 @@
+# In e2e test, this will not be used to deploy the app to test cluster.
+# This is created for testing purpose in order to deploy this app using kubectl
+# before writing e2e test.
+kind: Service
+apiVersion: v1
+metadata:
+  name: service-invocation
+  labels:
+    testapp: service-invocation
+spec:
+  selector:
+    testapp: service-invocation
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 3000
+  type: LoadBalancer
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: service-invocation
+  labels:
+    testapp: service-invocation
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      testapp: service-invocation
+  template:
+    metadata:
+      labels:
+        testapp: service-invocation
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/id: "myapp"
+        dapr.io/port: "3000"
+    spec:
+      containers:
+      - name: service-invocation
+        image: docker.io/[YOUR ALIAS]/e2e-service_invocation:dev
```

Comment:
Is this one for http and the other one for grpc? If so, I would expect this one to contain "http" just like the other image contains "grpc".

Reply:
I just used names matching the existing app which I added to.  This app now has "both", though for grpc it only has a client, it doesn't listen on grpc.